torch/csrc/jit/python/pybind_utils.h

```diff
@@ -240,6 +240,68 @@ struct VISIBILITY_HIDDEN PythonFutureWrapper
   }
 };
 
+struct VISIBILITY_HIDDEN PythonAwaitWrapper
```

Comment:
can you add a comment here describing what this does? (i.e. in scripted fns, jit.Await works on jit-scripted callable; but for eager mode you add this wrapper)